src/scripts/store/dimStoreItem.directive.template.html

```diff
@@ -30,6 +30,7 @@
          width="44" />
   </div>
   <div ng-class="::vm.badgeClassNames"
+       class="{{ vm.item.dtrRating || vm.item.quality ? 'item-stat-no-bg':'' }}"
```

Comment:
The problem is we don't get the dtrRating until after the promise resolves which is after this has been bound-once. Thus the class is never applied.